otcextensions/osclient/dis/v2/dis_utils.py

```diff
@@ -0,0 +1,124 @@
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
```

Comment:
maybe better to move this in some common package?

Reply:
Moved to otcextensions.common.cli_utils